apps/docs/content/components/Checkbox/index.mdx

```diff
@@ -0,0 +1,103 @@
+---
+title: Checkbox
+description: Use the checkbox component to select one or more options of a limited number of choices.
+---
+
+import ComponentLayout from '../../../src/layouts/component-layout'
+export default ComponentLayout
+
+## Anatomy
+
+![An image displaying a checkbox with a checkbox and label.](https://github.com/primer/brand/assets/131988618/2152c720-5ba8-4832-b85b-0d54a8450207)
+
+- Checkbox: A selection control.
+- Label: A description for the checkbox.
+
+## Usage
+
+Use checkboxes to allow users to select multiple items from a list of individual items, or to mark one individual item as selected. Unlike the [Radio](/components/Radio) component or [Select](/components/Select) component, Checkbox does not restrict users to choosing just a single option.
+
+Checkbox labels should be descriptive and concise. Write labels in sentence case.
+
+<DoDontContainer>
+  <Do>
+    <img src="https://github.com/primer/brand/assets/131988618/45e4877b-345c-4a3d-904f-8478e5b8f9a4" />
+    <Caption>Checkbox labels should be descriptive and concise.</Caption>
+  </Do>
+  <Dont>
+    <img src="https://github.com/primer/brand/assets/131988618/b97d2f8b-4f60-44d4-b028-fd0981a7751a" />
+    <Caption>
+      Don't make checkbox labels confusing, indirect, and avoid negative
+      statements.
+    </Caption>
+  </Dont>
+</DoDontContainer>
+
+<DoDontContainer>
+  <Do>
+    <img src="https://github.com/primer/brand/assets/131988618/8e29efd4-124f-4dc9-83c7-d80fd549c8da" />
+    <Caption>
+      Use consistent styling (starting with capitalization and not ending with
+      punctuation).
+    </Caption>
+  </Do>
+  <Dont>
+    <img src="https://github.com/primer/brand/assets/131988618/5d1c9068-cca5-484f-8573-047d4794f345" />
+    <Caption>
+      Don't have inconsistent styling (not starting with capitalization or
+      ending in punctuation).
+    </Caption>
+  </Dont>
+</DoDontContainer>
+
+<DoDontContainer>
+  <Do>
+    <img src="https://github.com/primer/brand/assets/131988618/12c7391f-ea66-4aac-bd32-25cf49c75c9d" />
+    <Caption>
+      Use first-person when asking users for acknowledgement or consent.
+    </Caption>
+  </Do>
+  <Dont>
+    <img src="https://github.com/primer/brand/assets/131988618/7e379318-3d12-4c99-b4ae-c33115791216" />
+    <Caption>
+      Don't use third-person when asking users for acknowledgement or consent.
+    </Caption>
+  </Dont>
+</DoDontContainer>
+
+<DoDontContainer>
+  <Do>
+    <img src="https://github.com/primer/brand/assets/131988618/d68ff8bc-a8ba-4413-8d06-91a9102ee2d2" />
```

Comment:
```suggestion
    <img src="https://github.com/primer/brand/assets/131988618/3a0a0eb1-372c-499b-9325-68337cef3428" />
```

Reply:
I think something weird is happening (maybe its me?) but the updated image is empty too 😟

Reply:
I fixed the image and updated the rest with 2x size for retina displays. This is ready to 🚢 !